docker-compose.yml

```diff
@@ -1,17 +1,17 @@
-# version: "3.7"
+version: "3.7"
 
-# services:
-#   web:
-#     build: .
-#     environment:
-#       NODE_ENV:
-#       MONGODB_URI: mongodb://mongo:27017/nodegoat
-#     command: sh -c "until nc -z -w 2 mongo 27017 && echo 'mongo is ready for connections' && node artifacts/db-reset.js && npm start; do sleep 2; done"
-#     ports:
-#       - "4000:4000"
+services:
+  web:
+    build: .
+    environment:
+      NODE_ENV:
+      MONGODB_URI: mongodb://mongo:27017/nodegoat
+    command: sh -c "until nc -z -w 2 mongo 27017 && echo 'mongo is ready for connections' && node artifacts/db-reset.js && npm start; do sleep 2; done"
+    ports:
+      - "4000:4000"
 
-#   mongo:
-#     image: mongo:4.4
-#     user: mongodb
-#     expose:
-#       - 27017
+  mongo:
+    image: mongo:4.4
```

Comment:
#### Automated PR Comment From coverity
<!-- 520e2455b7e091e28bcbe89ac67e9e89 -->
<!-- Comment managed by Git Commenter, do not modify! -->
### Coverity Issue - Container allows filesystem write
_Low CWE-552_
The docker service container is configured to permit writing to the root filesystem. This makes some security attack vectors such as privilege escalation, denial-of-service or authorization bypass possible since the container instance's filesystem can be tampered with.
### How to fix
Explicitly set the `read-only` attribute of the service to `true` to create a service container with a read-only filesystem.